lib/extractStyles.js

```diff
@@ -144,6 +144,34 @@ function extractStyles(src, staticNamespace, getClassNameAndComment) {
     var comment = (
       classNameAndComment.comment ? '/* ' + classNameAndComment.comment + ' */\n  ' : ''
     );
+
+    var hoverStyles = {};
+    var focusStyles = {};
+    var activeStyles = {};
+
+    // Check entry for any props with pseudo selector prefixes, move them to
+    // respective pseudo style object
+    Object.keys(entry.staticAttributes).forEach(function(key) {
+      if (key.indexOf('hover') === 0) {
```

Comment:
Seems like we should share this code with the other file using a new function `explodePseudoselectors(styles) -> {styles, hoverStyles, focusStyles, activeStyles}`